homeassistant/components/overkiz/button.py

```diff
@@ -27,20 +27,20 @@ class OverkizButtonDescription(ButtonEntityDescription):
     # My Position (cover, light)
     OverkizButtonDescription(
         key="my",
-        name="My Position",
+        name="My position",
```

Comment:
Why not make this "Position"?

Reply:
This is the name given by Somfy for the favorite position. "My" is written on their physical switch and within their application.